posix.mak

```diff
@@ -282,6 +282,7 @@ DDOC_VARS_PRERELEASE_VERBATIM=$(DDOC_VARS_PRERELEASE) \
 ################################################################################
 
 DDOC_BIN:=$G/ddoc
+DDOC_BIN_DMD:=$(DDOC_BIN) --compiler=$(DMD)
```

Comment:
Not sure where `:=` became a thing in our makefiles. We've kept it for special cases so far.

Reply:
Since we started to have ugly, hard-to-trace bugs due accidentally overwritten of variables, see e.g. https://github.com/dlang/dlang.org/pull/1953